exonum-java-binding/qa-service/src/main/java/com/exonum/binding/qaservice/QaServiceImpl.java

```diff
@@ -115,6 +122,15 @@ public void initialize(Fork fork, Configuration configuration) {
     createCounter(AFTER_COMMIT_COUNTER_NAME, fork);
   }
 
+  //TODO: add @Override when 4133 will be ready
+  public void resume(Fork fork, byte[] arguments) {
+    QaResumeArguments resumeArguments = parseResumeArguments(arguments);
+
+    checkExecution(!resumeArguments.getShouldThrowException(), RESUME_SERVICE_ERROR.code);
+
+    createCounter(RESUME_COUNTER_NAME, fork);
```

Comment:
It might be sensible to test multiple stop/resume operations with a single service. However, createCounter with a constant name will make subsequent 'resumes' fail. Do you think we can make something as simple, but supporting several resumes (e.g., with a configurable name)?